pandas/tests/frame/methods/test_sort_values.py

```diff
@@ -497,3 +516,87 @@ def test_sort_values_ignore_index(
 
         tm.assert_frame_equal(result_df, expected)
         tm.assert_frame_equal(df, DataFrame(original_dict))
+
+
+# class TestDataFrameSortKey:  # test key sorting (issue 27237), not yet implemented
```

Comment:
instead of commenting out can you just xfail them and refer to the issue number

Reply:
done.